tools/test_repair.sh

```diff
@@ -140,17 +140,14 @@ for (( i = 0; i < 100; i += 1 )); do
     (( generation += 1))
 
     echo ""
-    # Stop --lossy downstairs so it can't complete all its IOs
-    if [[ $choice -eq 0 ]]; then
-        kill "$ds0_pid"
-        wait "$ds0_pid" || true
-    elif [[ $choice -eq 1 ]]; then
-        kill "$ds1_pid"
-        wait "$ds1_pid" || true
-    else
-        kill "$ds2_pid"
-        wait "$ds2_pid" || true
-    fi
+
+    # Stop everything before dump so dump can access the extent metadata
```

Comment:
The previous comment's mention of not allowing IOs to complete is good context here, that should be kept.

Reply:
For a "--lossy" downstairs, the intent was that that downstairs was doing IO slower
than the other three downstairs.  Slow enough that indeed it had not completed all
the IOs that the other two downstairs had completed.  This functionality is important
for the test as we are testing a repair here, so to perform a repair, there must be some
difference between the three downstairs :)

Reply:
i thought the point of lossy was that it would straight up drop the IOs rather than just do them slower?

Reply:
Perhaps poorly named, yeah.
No, we don't drop any IOs, but the downstairs will "skip over" some as it walks the work list
It will also add a tiny delay into completing them.  So, eventually the IOs will finish, but it
will take longer than the other downstairs.